src/workerd/api/global-scope.c++

```diff
@@ -438,6 +438,20 @@ kj::Promise<WorkerInterface::AlarmResult> ServiceWorkerGlobalScope::runAlarm(
   }
 }
 
+jsg::Promise<void> ServiceWorkerGlobalScope::test(
+    Worker::Lock& lock, kj::Maybe<ExportedHandler&> exportedHandler) {
+  // TODO(someday): For Service Workers syntax, do we want addEventListener("test")? Not supporting
+  //   it for now.
```

Comment:
I would be in favor of just always requiring service workers syntax here.

Reply:
You mean always requiring modules syntax?

Reply:
Doh, yes, that.